npm-shrinkwrap.json

```diff
@@ -2,197 +2,22 @@
   "name": "basketball-gm",
```

Comment:
All the missing lines are probably because you ran `npm shrinkwrap` instead of `npm shrinkwrap --dev`. Please do the latter.